code/datums/martial/nemesis_kick.dm

```diff
@@ -0,0 +1,79 @@
+
+	var/atk_verb = pick("left hook","right hook","straight punch")
+
+	var/damage = rand(5, 10) + species.punchdamagelow //Nice stamina damage, especially considering the fact that we have 25% punch cooldown
+	if(!damage)
+		playsound(D.loc, species.miss_sound, 25, TRUE, -1)
+		D.visible_message("<span class='warning'>[A]'s [atk_verb] misses [D]!</span>", \
+						"<span class='danger'>You avoid [A]'s [atk_verb]!</span>", "<span class='hear'>You hear a swoosh!</span>", COMBAT_MESSAGE_RANGE, A)
+		to_chat(A, "<span class='warning'>Your [atk_verb] misses [D]!</span>")
+		log_combat(A, D, "attempted to hit", atk_verb)
+		return FALSE
+
+
+	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))
+	var/armor_block = D.run_armor_check(affecting, MELEE)
+
+	playsound(D.loc, species.attack_sound, 25, TRUE, -1)
+
+	D.visible_message("<span class='danger'>[A] [atk_verb]ed [D]!</span>", \
+					"<span class='userdanger'>You're [atk_verb]ed by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	to_chat(A, "<span class='danger'>You [atk_verb]ed [D]!</span>")
+
+	D.apply_damage(damage, STAMINA, affecting, armor_block)
+	log_combat(A, D, "punched (nemesis kick) ")
+	if(A != D)
+		gloves.gain_charge()
+	return TRUE
+
+/datum/martial_art/nemesis/disarm_act(mob/living/carbon/human/A, mob/living/D)
+	var/obj/item/clothing/gloves/rapid/nemesis/gloves = A.get_item_by_slot(ITEM_SLOT_GLOVES)
+	if(!gloves || !istype(gloves))
+		return FALSE
+
+	if(gloves.charge < NEMESIS_CHARGE_PER_KNOCKDOWN) //Common shoves if you don't have the charge
+		return FALSE
+
+	if(gloves.charge >= NEMESIS_CHARGE_PER_PARALYZE && D.body_position == LYING_DOWN) //Knockdown first and only then paralyzis.
+		D.visible_message("<span class='danger'>[A] finishes [D] with an overcharged glove, knocking [D.p_them()] out!</span>", \
+						"<span class='userdanger'>You're punched with an overcharged glove by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", null, A)
+		to_chat(A, "<span class='danger'>You overcharge your gloves and punch [D] with all of your might, finishing them!</span>")
+		playsound(get_turf(A), 'sound/weapons/egloves.ogg', 50, TRUE, -1)
+		D.Paralyze(NEMESIS_PARALYZIS_LENGTH)
+		gloves.lose_charge(NEMESIS_CHARGE_PER_PARALYZE)
+		return TRUE
+
+	D.visible_message("<span class='danger'>[A] punches [D] with [gloves], knocking them down!</span>", \
+						"<span class='userdanger'>You're punched with [gloves] by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", null, A)
+	to_chat(A, "<span class='danger'>You punch [D] using [gloves], knocking them down!</span>")
+	playsound(get_turf(A), 'sound/weapons/egloves.ogg', 50, TRUE, -1)
+	D.Knockdown(NEMESIS_KNOCKDOWN_LENGTH)
+	D.Jitter(NEMESIS_KNOCKDOWN_LENGTH)
+	gloves.lose_charge(NEMESIS_CHARGE_PER_KNOCKDOWN)
+	return TRUE
```

Comment:
holy fuck this is broken as shit

you do know that gloves of the north star work on disarm attacks too if you're on combat mode, right? you can just right click on somoene to knock them down, wait only _0.2 seconds_, then right click on them again to paralyze them for 6 seconds, which is more than enough time to cuff them. congratulations, you've managed to somehow make something even more broken than scooter+resting CQC leg sweep spam.

hell, even if the nemesis gloves didn't decrease the click cooldown of this, this is still an undisarmable baton with no cooldown (aside from the standard 0.8 second melee attack click cooldown). it's even more "balanced" if you lead with a flashbang or something to knockdown your opponent for a good long while first.

Reply:
Ooof, thanks for that.